virt/virt-2-6-release-notes.adoc

```diff
@@ -75,108 +65,95 @@ The SVVP Certification applies to:
-
-//The issues below are from the 2.5 release. See items above for new known issues for 2.6.
-//The below issues have not yet been pruned/updated for 2.6.
-
-* When a node fails in user-provisioned installations of {product-title} on bare metal deployments, the virtual machine does not automatically restart on another node. Automatic restart is supported only for installer-provisioned installations that have machine health checks enabled. Learn more about xref:/../virt/install/preparing-cluster-for-virt#preparing-cluster-for-virt[configuring your cluster for {VirtProductName}].
-
-* If you xref:../virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc#virt-creating-vddk-image_virt-importing-vmware-vm[add a VMware Virtual Disk Development Kit (VDDK) image] to the `openshift-cnv/v2v-vmware` config map by using the web console, a *Managed resource* error message displays. You can safely ignore this error. Save the config map by clicking *Save*. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1884538[*BZ#1884538*])
+* If you use a Fedora image as a boot source, it is no longer attached to a template if the PVC that you used to attach the boot source was previously provisioned. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1907187[*BZ#1907187*]) (link:https://bugzilla.redhat.com/show_bug.cgi?id=1913352[*BZ#1913352*])
 
-* When nodes are evicted, for example, when they are placed in maintenance mode during an {product-title} cluster upgrade, virtual machines are migrated twice instead of just once. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1888790[*BZ#1888790*])
+** As a workaround, attach a new PVC with the name `fedora` to a template before using it to create virtual machines from boot sources.
 
-//The issues below are from the 2.5 release. See items above for new known issues for 2.6.
+* If your {product-title} cluster uses OVN-Kubernetes as the default Container Network Interface (CNI) provider, you cannot attach a Linux bridge or bonding to the default interface of a host because of a change in the host network topology of OVN-Kubernetes. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1885605[*BZ#1885605*])
 
-* If your {product-title} cluster uses OVN-Kubernetes as the default Container Network Interface (CNI) provider, you cannot attach a Linux bridge or bonding to the default interface of a host because of a change in the host network topology of OVN-Kubernetes. As a workaround, you can use a secondary network interface connected to your host or switch to the OpenShift SDN default CNI provider. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1885605[*BZ#1885605*])
+** As a workaround, you can use a secondary network interface connected to your host or switch to the OpenShift SDN default CNI provider.
```

Comment:
```suggestion
** As a workaround, you can use a secondary network interface connected to your host, or switch to the OpenShift SDN default CNI provider.
```
nit but I raised this on the same sentence in the admonition in our docs - this throws me off because I hit the word 'switch' and think network switch, and the sentence then reads "you can use a secondary interface connected to a host or (network) switch .."